src/engine/SCons/Tool/msvc.xml

```diff
@@ -475,4 +475,38 @@ Valid values are '1' or '0'
 </summary>
 </cvar>
 
+<cvar name="VSWHERE">
+<summary>
+<para>
+Specify the location of vswhere.exe. 
+</para>
+
+<para>
+<literal>vswhere.exe</literal> is used to locate MSVC/MSVS installations.
+It was initially distributed with MSVS 2017. But can detect MSVC 2010 and newer with <literal>-legacy</literal> argument. 
```

Comment:
Propose:  The <command>vswhere</command> command is distributed with Microsoft Visual Studio and Build Tools since the 2017 edition, but is also available standalone. It fully detects 2015 and later editions, but with the <option>-legacy</option> argument, can detect installations of the 2010 through 2015 editions with limited data returned.